extensions/azdata/src/typings/azdata-ext.d.ts

```diff
@@ -283,13 +287,23 @@ declare module 'azdata-ext' {
 							memoryRequest?: string,
 							noWait?: boolean,
 						},
-						additionalEnvVars?: AdditionalEnvVars
+						additionalEnvVars?: AdditionalEnvVars,
+						loginSession?: AzdataLoginSession
 					): Promise<AzdataOutput<void>>
 				}
 			}
 		},
 		getPath(): Promise<string>,
-		login(endpoint: string, username: string, password: string, additionalEnvVars?: AdditionalEnvVars): Promise<AzdataOutput<any>>,
+		login(endpoint: string, username: string, password: string, additionalEnvVars?: AdditionalEnvVars): Promise<AzdataOutput<void>>,
+		/**
+		 * Acquires a session for the specified controller, which will log in to the specified controller and then block all other commands
```

Comment:
`then block all other commands from executing` is not accurate, it should be `blocking a new session from being created until...`